build_bundle_image.sh

```diff
@@ -145,8 +145,20 @@ function check_usage {
 		print_help
 	fi
 
-	if [[ -n "${LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES}" ]] &&
-	! [[ "${LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES}" =~ \[(\"(http|https):\/\/[-\d\w]+\:[\d]+\")+(\,)*(\s)*(\"(http|https):\/\/[-\d\w]+\:[\d]+\")*\] ]]
+	if [[ -n "${LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES}" ]] && \
+	   ! ( echo "${LIFERAY_DOCKER_ELASTICSEARCH_NETWORK_ADDRESSES}" | grep -qP "\[?(\"?(http|https):\/\/[.\w-]+:[0-9]+\"?)+(,\s*\"(http|https):\/\/[.\w-]+:[0-9]+\")*\]?" )
```

Comment:
@kiwm, please remove the `\`, use the full name of the `grep` parameters and use the `\d` in the regex if it's feasible.